src/e2e-tests/page-objects/smartthings/smartthings.component.js

```diff
@@ -0,0 +1,70 @@
+// smartthings.component.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+import TheaterTabPage from './theaterTab.page';
+import OfficeTabPage from './officeTab.page';
+import BackRoomTabPage from './backRoomTab.page';
+import BedroomTabPage from './bedroomTab.page';
+import KitchenTabPage from './kitchenTab.page';
+import BathroomTabPage from './bathroomTab.page';
+import OutsideTabPage from './outsideTab.page';
+
+class SmartThingsComponent extends Page {
```

Comment:
This is a page that shall contain, using composition, all the components you described above

Reply:
Updated.